packages/http-client-java/generator/http-client-generator-core/src/main/java/com/microsoft/typespec/http/client/generator/core/mapper/ClientMethodMapper.java

```diff
@@ -808,12 +806,10 @@ private static void createPageableClientMethods(Operation operation, boolean isP
             .groupedParameterRequired(false)
             .methodVisibility(visibilityFunction.methodVisibility(false, defaultOverloadType, false));
 
-        if (settings.isGenerateAsyncMethods()) {
-            methods.add(builder.build());
+        methods.add(builder.build());
 
-            // overload for versioning
-            createOverloadForVersioning(isProtocolMethod, methods, builder, parameters);
-        }
```

Comment:
A minor fix around here.

I am not exactly sure why the code was written this way. But I think for all existing situations `isGenerateAsyncMethods=true`, except unbranded. So it is safe to switch, as unbranded only enabled pageable now.